src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java

```diff
@@ -2287,6 +2308,8 @@ public ObPayload execute(ObPair<Long, ObTableParam> obPair) throws Exception {
                 request.setTableId(tableParam.getTableId());
                 // partId/tabletId
                 request.setPartitionId(tableParam.getPartitionId());
+                request.getTableQueryAndMutate().setIsCheckAndExecute(checkAndExecute);
```

Comment:
What type of request is it? Besides, I can't find where the `getTableQueryAndMutate()` is.

Reply:
By the way, checkAndExecute should be set when a single operation is created. I guess we should set this flag when executing. However, it depends on you.